doc/Manual_testing_and_exploratory_testing_documentation.md

```diff
@@ -0,0 +1,220 @@
+ # Manuális tesztelés
+ 
+## Játékmenet alapfunkciók tesztelése
+
+### Játékos hozzáadása
+
+Teszt terv, játékos hozzáadásának megpróbálása ha:
+- a) nincs megadva név
+- b) nincs megadva karakter típus
+- c) van név és karakter kiválasztva
+- d) van már azonos nevű karakter
+
+Tesztek eredménye:
+
+**a)** ha nincs megadva név:
+
+![](img/feladat_3_Manualis_Test/jatekos_hozzaadas1.jpg)
+
+Végeredmény: Nem lett hozzáadva új karakter.
+
+**b)** ha nincs megadva karakter típus
+
+![](img/feladat_3_Manualis_Test/jatekos_hozzaadas2.jpg)
+
+Végeredmény: Nem lett hozzáadva új karakter.
+
+**c)** van név és karakter is kiválasztva
+
+![](img/feladat_3_Manualis_Test/jatekos_hozzaadas3.jpg)
+
+Végeredmény: hozzá lett adva az új játékos/karakter.
+
+**d)** van már azonos nevű játékos
+
+![](img/feladat_3_Manualis_Test/jatekos_hozzaadas4.jpg)
+![](img/feladat_3_Manualis_Test/jatekos_hozzaadas5.jpg)
+
+!! Lehetséges probléma !!
+
+Végeredmény: A második játékos létrehozásakor az első játékos neve eltűnt a lenti listából, helyére az új játékos került, hibáról nem tájékoztatott a program.
```

Comment:
Is it a feature or a bug?
Tudjuk, hogy miért történik ez?
Akarjuk javítani?

Reply:
Felfogható feature-nek, mivel a már hozzáadott játékosok módosítására nincs mód, ezért így lehet a karakter típust utólag módosítani